airflow/sensors/base_sensor_operator.py

```diff
@@ -58,24 +59,24 @@ class BaseSensorOperator(BaseOperator, SkipMixin):
     :type mode: str
     """
     ui_color = '#e6f1f2'
-    valid_modes = ['poke', 'reschedule']
+    valid_modes = ['poke', 'reschedule']  # type: Iterable[str]
 
     @apply_defaults
     def __init__(self,
-                 poke_interval=60,
-                 timeout=60 * 60 * 24 * 7,
-                 soft_fail=False,
-                 mode='poke',
+                 poke_interval: int = 60,
+                 timeout: int = 60 * 60 * 24 * 7,
```

Comment:
I think it would be better to use `float`. Because `Int < Float`.

Reply:
Good point. Thanks!